modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/DurableBackgroundCleanupIndexTreeTaskV2.java

```diff
@@ -278,6 +278,18 @@ public DurableBackgroundCleanupIndexTreeTaskV2() {
         return outFut;
     }
 
+    /**
+     * Renames index's trees.
+     *
+     * @param grpCtx Cache group context.
+     * @throws IgniteCheckedException If failed to rename index's trees.
+     */
+    public void renameIndexTrees(CacheGroupContext grpCtx) throws IgniteCheckedException {
```

Comment:
Please add a test for this method to **org.apache.ignite.internal.processors.cache.index.RenameIndexTreeTest**.

Reply:
I doubt that it's needed, because this method is not static hence it should be tested with `DurableBackgroundCleanupIndexTreeTaskV2` and this class is already tested in the IndexDropTest

Reply:
But we need to check that after calling this method, we will not rename the indexes after the task starts.